### DISCUSSION TITLE
Jetbrains Github Copilot and Copilot Chat Rendered Code Snippets are tiny in HiDPI mode on Linux

### DISCUSSION THREAD

**Discussion by spaceman7777** [community/community#77757] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi,
### Issue:
Github Copilot and Copilot chat are rendered near-unusable due to tiny font size of code snippets rendered in the addon side panels. The font size does not scale to HiDPI sizes. All of the explainer text renders fine though. It's just the code snippets.

### Platforms:
Linux, using Xfce in "retina-style" HiDPI mode (and likely any other desktop environment using X11, and potentially Wayland). Specific platforms I've observed the bug on are Ubuntu 22.04 and 23.10

### Potential Resolutions:
- **_Add a "code font size" setting (will fix this permanently, and add a new feature for everyone)_**
- Specifically target the scaling issue with the text

### Notes:
- This is likely related to markdown rendering
- Setting custom CSS in the (in-house) Jetbrains authored markdown addon does not resolve the issue on this platform, neither does disabling the markdown addon altogether. (This is a solution that has some documentation concerning it on the web. Unfortunately, it does not appear to work, as far as I can tell.)
- Disabling Jetbrains markdown support fixes the rendering size of _already rendered_ code snippets, switching them to a properly scaled font, however any new code snippets that are rendered, are back to the tiny font size

### Conclusion: 
This makes Github Copilot semi-unusable on the platform :(

### Example screenshots:
![Selection_1394](https://github.com/community/community/assets/122674880/ae2c00c3-1fc4-44f5-aeb1-33dba6bdbdb9)
![Selection_1395](https://github.com/community/community/assets/122674880/75b1016f-3ede-4a09-82a2-7c738280efd4)

**Comment by Akash1134** [comment]
Hi @spaceman7777 ,

I am not able to reproduce this issue at my end. However, I am dropping another [thread from JetBrains Youtrack](https://youtrack.jetbrains.com/issue/IDEA-339450/Tiny-Rendered-Markdown-Text-on-Linux) that looks related to this issue. Have you reviewed it and tried the suggestions offered? 

Meanwhile, I am going to check with my team internally to see if this is a known issue. 

Thanks!
